Cargo.toml

```diff
@@ -24,6 +24,7 @@ rocket = "0.4.0"
 rocket_contrib = { version = "0.4.0", features = ["json"] }
 rocket_i18n = "0.4.0"
 rpassword = "2.0"
+runtime-fmt = "0.3.0"
```

Comment:
what's this do?

Reply:
like format!() or println!(), but with the base string unknown at compile time (when we fetch the url, we get something like `https://<instance>/authorize_interaction?uri={uri}`, which we obviously can't no at compile time)